app/src/display/Kconfig

```diff
@@ -42,6 +41,11 @@ choice ZMK_DISPLAY_STATUS_SCREEN
 config ZMK_DISPLAY_STATUS_SCREEN_BUILT_IN
     bool "Built in status screen"
     select LV_OBJ_LABEL
+    imply LV_USE_THEME_MONO
+    imply ZMK_WIDGET_LAYER_STATUS
+    imply ZMK_WIDGET_BATTERY_STATUS
+    imply ZMK_WIDGET_OUTPUT_STATUS
+    imply ZMK_WIDGET_PERIPHERAL_STATUS
```

Comment:
Is the conditional logic we had before (BT enabled, central vs, peripheral) not necessary?

Reply:
As far as I can tell, the `depends on` was all we needed. Things seem to function equivalently before and after making this change. Tested both left and right with built-in and custom widgets.

Reply:
Oh right, we still have the `depends on` with the same conditions, I somehow missed that 😅